rpc/client_opt.go

```diff
@@ -34,7 +34,8 @@ type clientConfig struct {
 	httpAuth    HTTPAuth
 
 	// WebSocket options
-	wsDialer *websocket.Dialer
+	wsDialer           *websocket.Dialer
+	wsMessageSizeLimit *int64
```

Comment:
I think declare this variable as `int64` is sufficient, default 0 means:

1. no limit(not compatible with current version);
2. or the same to `wsMessageSizeLimit`

Reply:
That's why this is currently *int64. nil signifies nothing is set (use hard coded value), non-nil means this has been specified and should be applied.

Reply:
`-1` for no limit?

Reply:
The gorilla websocket library treats 0 as no limit, so it seems cleaner to try to match that in the settings here. Is there a problem using `nil` as the default value (matching the current behavior)?
I can change this if you feel strongly about it...

Reply:
So let's add a comment to say that 0 means no limit

Reply:
Good idea. I added a comment on the `WithWebsocketMessageSizeLimit` method I created to mention 0 means no limit.